src/components/postBoard.tsx

```diff
@@ -5,14 +5,40 @@ import Post from './post';
 
 class PostBoard extends Component {
 
+    state = {
```

Comment:
I'd recommend defining an interface or type for this state. Would just be an array of Post objects.